src/utils/storageFacade.utils.ts

```diff
@@ -349,5 +547,62 @@ async function updateVuln(cve: Vulnerability, sessionId: number) {
   return status;
 }
 
+/************* DASHBOARD HELPERS *******************/
+function mapPkgParamToColumn(sortParam: PackageViewParam): string {
```

Comment:
is paramater mapped here solely for sorting/filtering or are you setting the column so its the first column next to the name?